app/controllers/media_controller.rb

```diff
@@ -47,7 +58,31 @@ def stacks_media_stream_params
     allowed_params.slice(:id, :file_name, :format)
   end
 
+  def id
+    allowed_params[:id]
+  end
+
+  def file_name
+    allowed_params[:file_name]
+  end
+
   def load_media
     @media ||= StacksMediaStream.new(stacks_media_stream_params)
   end
+
+  def token_still_valid?(token_string, expected_id, expected_file_name, expected_user_ip_addr)
```

Comment:
and now I'm wondering if splitting this method out is better than just calling StacksMediaToken method inside `verify_token` method.   I'm okay with it split out like this, tho.